src/components/sidebar/ohri-form-sidebar.component.tsx

```diff
@@ -87,9 +87,13 @@ function OHRIFormSidebar({
           )
         );
       })}
-      <hr className={styles.sideBarHorizontalLine} />
+      {mode == 'view' ? (
+        <div className={styles.sideBarHorizontalLine}></div>
+      ) : (
+        <hr className={styles.sideBarHorizontalLine} />
+      )}
```

Comment:
```suggestion
      {mode !== 'view'  (
        <hr className={styles.sideBarHorizontalLine} />
      )}
```

Reply:
The sideBarHorizontalLine style is just a margin-top of 1.5rem. I added it to view mode because I thought it looked better when there was space between the page summary and the button. See attached images:
1. With the space
2. without.
If you still think 2 looks ok, then I can effect the change
![20230130_110228](https://user-images.githubusercontent.com/12844964/215421810-cde41bfb-8b29-453a-ae3d-73c377ce1d9e.png)
![20230130_110249](https://user-images.githubusercontent.com/12844964/215421844-4a855f83-dc60-48d6-aa9b-092567403a62.png)

Reply:
This needs to be fixed or aligned within the "Close" wrapper element or the pages element above. Not some dynamically added invisible div as a replacement of the divider.